src/main/java/io/openliberty/tools/intellij/lsp4mp4ij/psi/core/java/completion/IJavaCompletionParticipant.java

```diff
@@ -24,6 +24,7 @@
  * @author datho7561
  */
 public interface IJavaCompletionParticipant {
+	public static final ExtensionPointName<IJavaCompletionParticipant> EP_NAME = ExtensionPointName.create("open-liberty.intellij.javaCompletionParticipant");
```

Comment:
This extension point is unused. The real one is in `JavaCompletionDefinition`. I know this keeps coming back every time we synch from Quarkus Tools. We deleted it the last time. It's not harmful but it is misleading (potentially confusing) to someone reading the code.

Reply:
Ok, we should mark this and the other one as unused so this doesn't happen again.

Reply:
I'm not sure what you mean by marking it, but I guess this could be something we include in our process for synching up lsp4mp4ij. It's a change we made in LTI to support a common code path for both MP and Jakarta EE completion.